### PR TITLE
Upgrade to Tensorflow 2.6

### PR DESCRIPTION
**Pull request by Philmod**
Now that the COS image has the NVIDIA Drivers R470, upgrade Tensorflow.

http://b/177304566

### CODE REVIEW

**Comment by Philmod** [comment]
Unfortunately Torch doesn't seem to recognize CUDA 11.4, asked on Github: https://github.com/mlverse/torch/issues/807

**Comment by Philmod** [comment]
The Torch installation works, but the Torch GPU test fails when it's run with all the other tests (it passes if run alone): 

```
Error (test_torch.R:15:3): gpu imports
<Rcpp::exception/C++Error/error/condition>
Error in `torch_tensor_cpp(data, dtype, device, requires_grad, pin_memory)`: CUDA error: out of memory
```

**Comment by dfalbel** [comment]
> The Torch installation works, but the Torch GPU test fails when it's run with all the other tests (it passes if run alone):
> 
> ```
> Error (test_torch.R:15:3): gpu imports
> <Rcpp::exception/C++Error/error/condition>
> Error in `torch_tensor_cpp(data, dtype, device, requires_grad, pin_memory)`: CUDA error: out of memory
> ```

This probably happens because TensorFlow allocates the entire GPU memory and only gives it back when R is closed. It seems that you could set `TF_FORCE_GPU_ALLOW_GROWTH` to `true` as per [here](https://stackoverflow.com/a/55541385/3297472).

File: test [hunk 1]
```diff
@@ -87,8 +87,12 @@ docker kill jupyter_test_r && docker rm jupyter_test_r
 docker run --rm --name=papermill_test_r --read-only --net=none \
     "$IMAGE_TAG" python -c 'import sys;import papermill as pm; print(pm.__version__)'
 
+
+# TF_FORCE_GPU_ALLOW_GROWTH is to prevent tensorflow from allocating the totality of a GPU memory.
```
Comment (rosbo) [inline test:91]: 👍  We do the same for our Python image: https://github.com/Kaggle/docker-python/blob/ce2a5245a1e2481e7384caec03feafe38bac24da/test#L105

**Review by rosbo — APPROVED** [review]

File: test (hunk 1 above)
Comment (Philmod) [reply test:91]: Oh nice, I wasn't aware of that!

**Pull request merged**